context/app/static/js/components/Search/DownloadButton/DownloadButton.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+
+import { StyledButton } from './style';
+
+function DownloadButton(props) {
+  const { type } = props;
+  const lcPluralType = `${type.toLowerCase()}s`;
+  return (
+    <SecondaryBackgroundTooltip title="Download a TSV of the table metadata. This will not take into account any filtering done on the search page.">
+      <StyledButton href={`/api/v0/${lcPluralType}.tsv`} target="_blank" component="a">
```

Comment:
Should we add a google analytics event for this?

Reply:
Will do: Doesn't hurt anything.